Automattic/wp-calypso
# Plans: Add plan features to Plan Overview page

This pull request addresses part of https://github.com/Automattic/wp-calypso/issues/1151 by adding a list of plan features at the bottom of the `Plan Overview` page. It also displays a nudge to upgrade to Business when the current plan on trial is Premium:

![screenshot](https://cloud.githubusercontent.com/assets/594356/11791031/fd8691a8-a26c-11e5-9329-a0504d689b75.png)
 
#### Testing instructions

1. Run `git checkout add/plan-overview-features` and start your server
2. Get a new test account and blog
3. Enable your sandbox and proxy
4. Select a plan by clicking the `Start Free Trial` button on the [`Plans` page](http://calypso.localhost:3000/plans)
5. Click the `Start 14 Day Free Trial` button on the `Secure Payment` page
6. Check that the purchase was successful
7. Navigate back to the `Plans` page which should now display the `Plan Overview` variant
8. Check that all buttons as well as the upgrade link work fine

#### Reviews
